### PR TITLE
Fix Barbolani's momentary lapse of sanity (swoopShutUp)

### PR DESCRIPTION
**Pull request by SonorousMeerkat**
Replaces the very strangely named swoopShutUp with diveGear, a more relevant name considering this array is for adding diving gear to seaport boxes.

### CODE REVIEW

File: A3-Antistasi/Templates/Occupants3CBBAFT.sqf [hunk 1]
```diff
@@ -15,6 +15,22 @@ NATOMortar = "UK3CB_BAF_Static_L16_Deployed_Tropical_RM";
 staticATOccupants = "RHS_TOW_TriPod_USMC_WD";
 staticAAOccupants = "RHS_Stinger_AA_pod_WD";
 
+//NATO PvP Loadouts
```
Comment (Toakan) [inline A3-Antistasi/Templates/Occupants3CBBAFT.sqf]: Did you mean to include this file?

Comment (Toakan) [inline A3-Antistasi/Templates/Occupants3CBBAFT.sqf]: Did you mean to include this file?

Comment (SonorousMeerkat) [reply A3-Antistasi/Templates/Occupants3CBBAFT.sqf]: I won't ever go away.

Comment (Toakan) [reply A3-Antistasi/Templates/Occupants3CBBAFT.sqf]: It irks me that it took 15 pushes to get this PR done 🥇

**Comment by SonorousMeerkat** [comment]
Great, now it has the entire initVar thing conflicting...

**Comment by SonorousMeerkat** [comment]
Oh... the initVar thing means my entire change needs fixing -.-
WONDERFUL

**Comment by SonorousMeerkat** [comment]
DONE! Ready for looking at!

**Pull request merged**